packages/bierzo-wallet/src/routes/payment/components/CurrencyToSend/index.tsx

```diff
@@ -33,7 +35,7 @@ const useStyles = makeStyles((theme: Theme) => ({
     color: '#ffffff',
   },
 
-  sendLabel: {
+  sendLabelRoot: {
```

Comment:
I do not like the `Root` suffix, when you add the class to mui's component like:
```
const sendLabelClasses = {
    root: classes.sendLabelRoot,
  };
```
You directly connect the intentions. No need to add extra suffix  😉